ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -53,7 +53,7 @@ The following data model keys have been removed from `eos_designs` in v5.0.0.
 | Removed key | New key |
 | ----------- | ------- |
 | old key 1(cvp_instance_ip) | new key(TODO) |
-| old key 2(defs_adapter_config) | new key(TODO) |
+| adapter_config.port_channel.short_esi | adapter_config.port_channel.ethernet_segment.short_esi |
```

Comment:
I think, the respective data-models should be mentioned here, like -
`connected_endpoints.short_esi | ethernet_segment.short_esi`

same for `network_ports` and `port_profiles`

Reply:
@ClausHolbechArista

Reply:
Yes please be elaborate so someone searching will find it.

Reply:
You also missed the port_channel part here.
```suggestion
| network_ports[].port_channel.short_esi | network_ports[].ethernet_segment.short_esi |
| <connected_endpoints_key>[].adapters[].port_channel.short_esi | <connected_endpoints_key>[].adapters[].ethernet_segment.short_esi |
| port_profiles[].port_channel.short_esi | port_profiles[].ethernet_segment.short_esi |
```

Reply:
Fixed